core/controllers/base.py

```diff
@@ -344,7 +344,7 @@ def render_template(
                 app_identity_services.get_gcs_resource_bucket_name()),
             # The 'path' variable starts with a forward slash.
             'FULL_URL': '%s://%s%s' % (scheme, netloc, path),
-            'SITE_FEEDBACK_FORM_URL': feconf.SITE_FEEDBACK_FORM_URL,
+            'SITE_FEEDBACK_FORM_URL': constants.SITE_FEEDBACK_FORM_URL,
```

Comment:
Since this is used to insert values into `base.html` and we don't have `SITE_FEEDBACK_FORM_URL` in `base.html` anymore you can remove this whole line.

Reply:
Okay, Thanks

Reply:
I noticed that in `core/controllers/base_test.py`, there's a test for auto_escaping jinja templates.

Since I've removed `SITE_FEEDBACK_FORM_URL` from `core/controllers/base_test.py`, it's making the test to fail. Should the test also be removed knowing that `constants.SITE_FEEDBACK_FORM_URL` is actually from the `constants.js` file?

Also, while reviewing the previous work that was done for `DEV_MODE`, I noticed that it was left in `core/controllers/base_test.py`, while also getting its value from `constants.js`

Reply:
You can probably try to use a different constant in the auto-escaping test.

There are multiple constants that are in `/core/tests/build_sources/pages/base.html` but not in `/templates/pages/base.html`, you can remove them in different PR if you want.

Reply:
Okay Done